Google AdWords documentation update (#390)
rudderlabs/rudderstack-docs: cloud-extract-sources/google-adwords.mdx

<div class="infoBlock">

All the Cloud Extract sources support sending data only to a <a href="https://www.rudderstack.com/docs/data-warehouse-integrations/">warehouse destination</a>.
</div>

## Getting started

To add Google AdWords as a source in your [RudderStack dashboard](https://app.rudderstack.com), follow these steps:

1. In the left panel, go to **Directory**. Under **Sources**, click on **Cloud Extract** and select **Google Adwords**.

<img src="../assets/cloud-extract-sources/google-adwords-dashboard.png" alt="Google AdWords Cloud Extract source" />

2. Assign a name to your source, and click on **Next**.

3. Click on **Connect with Google AdWords** and give RudderStack the required access permissions. Then, click on **Next**.

<div class="infoBlock">

If you have already connected RudderStack to your Google AdWords account, your credentials should appear automatically under <strong>Use existing credentials</strong>.
</div>

### Source settings

- **Customer ID**: After you connect RudderStack to your Google AdWords account, the associated AdWords customer IDs will automatically appear in this dropdown. 

> - **Start date**: This option specifies the earliest date when RudderStack will import all your historical AdWords data.

Reviewer: ```suggestion
- **Start date**: This option specifies the earliest date by when RudderStack can import all your historical AdWords data.
```

<div class="infoBlock">

Note that the <strong>Start Date</strong> setting is valid only for the first (historical) sync.
</div>

- **Conversion window**: Select your [AdWords conversion window](https://support.google.com/google-ads/answer/3123169?hl=en) from the dropdown options.

<img src="../assets/cloud-extract-sources/google-adwords-source-settings.png" alt="AdWords source settings" />

### Destination settings and data update schedule

> - **Table prefix**: RudderStack will create a table in your data warehouse with this prefix name and load all your AdWords data into it.

Reviewer: ```suggestion
- **Table prefix**: Creates a table in your data warehouse with the specified prefix name and loads all your AdWords data into it.
```

- **Run frequency**: Select the frequency of data import from your AdWords account to RudderStack. 

- **Sync starting at**: Select the time when you want RudderStack to sync the data from your AdWords account.

<img src="../assets/cloud-extract-sources/google-adwords-destination-settings.png" alt="AdWords destination settings" />

### Specifying the data to import

- Finally, specify the AdWords data you want RudderStack to import. By default, all the resources are selected.

<img src="../assets/cloud-extract-sources/google-adwords-resources.png" alt="AdWords resources to import" />

- You can also import a custom report as a resource. Refer to the [Adding a custom report as resource](#adding-custom-reports-as-resources) section below for more information.

> That's it! Google AdWords is now successfully configured as a source. RudderStack will start importing the Google AdWords data as per the specified frequency. 

Reviewer: ```suggestion
Google AdWords is now successfully configured as a source. RudderStack will start importing the Google AdWords data as per the specified frequency. 
```

> You can further connect this source to your data warehouse by clicking on **Connect Destinations** or **Add Destinations**, as shown:

Reviewer: ```suggestion
You can further connect this source to your data warehouse by clicking on **Add Destination**, as shown:
```

<img src="../assets/cloud-extract-sources/google-adwords-connect-destinations.png" alt="Connecting a destination to AdWords source" />

<div class="successBlock">

>   Use the <strong>Connect Destinations</strong> option if you have already configured a data warehouse destination in RudderStack/ To configure a warehouse destination from scratch, click on the <strong>Add Destination</strong> button.

Reviewer: Can't locate 'Connect Destinations' option.

Author: fixed this.

| Campaigns                     | `campaigns`                    | Contains information about the campaigns in your AdWords account. The columns in this table are `name`, `startDate`, and `frequencyCap` with `id` as the primary key.  |
| Report: Keywords Performance  | `keywords_performance_report`  | Contains information about your AdWords keywords' performance. The columns in this table are `ActiveViewImpressions`, `AverageCost`, `AverageCpc`, `CampaignName`, and `KeywordMatchType`.    |
| Report: Click Performance     | `click_performance_report`     | Contains information about your AdWords' click performance. The columns in this table are `AccountDescriptiveName`, `AdGroupStatus`, `CampaignId`, `Clicks`, and `ClickType`.                           |
| Report: Placement Performance | `placement_performance_report` | Contains the your AdWords account's placement performance. The columns in this table are `AverageCpc`, `AdGroupStatus`, `Clicks`, `Device`, and `Ctr`.                                         |

> ## FAQs

Reviewer: ```suggestion
## FAQ
```
